### DISCUSSION TITLE
Save settings

### DISCUSSION THREAD

**Discussion by ccdffddfddfdsfedeee** [RedShyGuy/Vapecord-ACNL-Plugin#34] [Q&A]
Is there a way to save settings so you do not have to set them every time you open the game?

**Comment by RedShyGuy** [comment]
Yea, in Tools -> Settings should be a check to save them

**Reply by ccdffddfddfdsfedeee** [reply]
How do you teleport all the players into the island building?

**Reply by SirEnder125** [reply]
I believe that function was removed in the later versions. (After 1.8.2?)

**Reply by RedShyGuy** [reply]
Its in Room Teleport

**Reply by ccdffddfddfdsfedeee** [reply]
Is there a way to fix freezing when opening menus and it needs to be restarted?

**Reply by RedShyGuy** [reply]
It shouldnt freeze, when does it freeze?

**Reply by ccdffddfddfdsfedeee** [reply]
I will try updating, also do you have a list of sound ids?
